deploy/proxy.env

```diff
@@ -21,3 +21,4 @@ SESSION_STORE_SECRET=wheeee!
 CONSOLE_PROXY_CERT_KEY=use local dev-cert/pproxy.key in portal-proxy repo
 CONSOLE_PROXY_CERT=use local dev-cert/pproxy.crt in portal-proxy repo
 ENCRYPTION_KEY=B374A26A71490437AA024E4FADD5B497FDFF1A8EA6FF12F6FB65AF2720B59CCF
+LOG_LEVEL=debug
```

Comment:
This will mean any docker compose deployment will have debug logging. Probably want to set this to info.

Should look to see if we can pass this as an env when running standupdevenv

Reply:
That was the intent, since docker-compose environments are dev environments